src/main/app/src/Components/HomeComponent/Home.tsx

```diff
@@ -0,0 +1,230 @@
+import React, {useState, useEffect, useContext, useCallback, ReactElement} from 'react';
+
+import {ILandscape} from '../../interfaces';
+import GenericModal from '../ModalComponent/GenericModal';
+import LandscapeLog from '../LandscapeComponent/Log/LandscapeLog';
+import Command from '../CommandComponent/Command';
+import {Link} from 'react-router-dom';
+import {createStyles, Theme, makeStyles} from '@material-ui/core/styles';
+import CommandContext from '../../Context/Command.context';
+import LandscapeContext from '../../Context/Landscape.context';
+import Typography from '@material-ui/core/Typography';
+import MenuIcon from '@material-ui/icons/Menu';
+import Grid from '@material-ui/core/Grid';
+import {Button, Fab} from '@material-ui/core';
+import './Home.scss';
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        marginBottom: {
+            marginBottom: theme.spacing(1),
+        },
+        extendedIcon: {
+            marginRight: theme.spacing(1),
+        },
+    }),
+);
+
+const Home: React.FC = () => {
+    const [modalContent, setModalContent] = useState<string | ReactElement | ReactElement[] | null>(
+        null
+    );
+    const [landscapes, setLandscapes] = useState<ILandscape[]>();
+    const [loadLandscapes, setLoadLandscapes] = useState<boolean>(true);
+
+    const commandContext = useContext(CommandContext);
+    const landscapeContext = useContext(LandscapeContext);
+    const classes = useStyles();
+    //Could be moved into useEffect but can be used for a reload button later on
+    const getLandscapes = useCallback(async () => {
+        if (loadLandscapes) {
+            await fetch(process.env.REACT_APP_BACKEND_URL + '/api/')
+                .then(response => {
+                    return response.json();
+                })
+                .then(json => {
+                    setLandscapes(json);
+                    setLoadLandscapes(false);
+                    landscapeContext.landscapes = json;
+                    commandContext.message = 'Loaded landscapes.';
+                });
+        }
+    }, [commandContext.message, landscapeContext.landscapes, loadLandscapes]);
+
+    //ComponentDidMount
+    useEffect(() => {
+        getLandscapes();
+    }, [getLandscapes]);
+
+    const enterLog = (l: ILandscape) => {
+        setModalContent(<LandscapeLog landscape={l}/>);
+        commandContext.message = 'Showing log: ' + l.identifier;
+    };
+
+    const enterLandscape = (l: ILandscape) => {
+        commandContext.message = 'Entering landscape: ' + l.identifier;
+    };
+    // Render
+    let content: string | ReactElement[] = 'Loading landscapes...';
+    if (landscapes) {
+
+        content = landscapes.map(l => {
+            return (
+
+
+                <Grid key={l.identifier} className={'landscapeContainer'} container spacing={3}>
+
+                    <Grid item xs={12} sm={12}>
```

Comment:
I remember having used this before myself. Did you check how it behaves when you scale the page? I guess you did :)

Reply:
no it doesnt work properly on scaling, i'll sit down with daniel tomorrow and will talk with him abit about material ui and our requirements and goals for our styling